### PR TITLE
Blog post for cs-reboot-info

### PR DESCRIPTION
**Pull request by briancurtin**
This post outlines our Cloud Servers reboot situation and introduces the cs-reboot-info tool. This was reviewed by Chris Kuehl.

### CODE REVIEW

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+author: Developer Experience Team
+published: true
+categories:
+    - devex
+    - utils
+---
+
+In the wake of
+[recently announced vulnerabilities](http://xenbits.xen.org/xsa/) to the
+Xen hypervisor that our Cloud Servers platform is built atop, a reboot
```
Comment (jnoller) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: s/atop/on top of

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+affected servers will be rebooted, your
+[https://mycloud.rackspace.com/](Cloud Control Panel) will contain the
+information for whichever region is currently visible. Alternatively, you
+can run our [`cs-reboot-info`](https://github.com/rackerlabs/cs-reboot-info)
+tool to discover the reboot windows of servers across all regions at once.
+Downloads for many platforms are available
+[here](https://github.com/rackerlabs/cs-reboot-info/#installation---binaries).
+
+When given a username and API key, which can be found in the *Account Settngs*
+section of our control panel, the utility will list the reboot windows
```
Comment (ycombinator) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Link to control panel?

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Additionally, a CSV file containing the details can be generated with
+the `--csv` option.
+
+    $ ./cs-reboot-info --csv <username> <api key>
+    Regions with a Cloud Servers endpoint: ORD, DFW, SYD, IAD, HKG
+    Found both First and Next Generation endpoints.
+    Writing output to cs-reboot-info-output.csv
+
+For more information, see [https://github.com/rackerlabs/cs-reboot-info]()
```
Comment (ycombinator) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: I'm not familiar with this linking syntax. Does it create a link whose label is the same as the href?

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+Generation Cloud Servers. The details of our announcement are available at
+[https://community.rackspace.com/general/f/53/t/4978]() and via
+[https://status.rackspace.com/]().
+
+In order to complete the patching of our systems, we have scheduled
+reboot windows on a per-region basis beginning Monday March 2 and running
+through Monday March 9. To discover the time ranges during which your
+affected servers will be rebooted, your
+[https://mycloud.rackspace.com/](Cloud Control Panel) will contain the
+information for whichever region is currently visible. Alternatively, you
```
Comment (jnoller) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: for the region you are currently viewing (note you can change this with the region selector on the left hand side of the control panel)

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+can run our [`cs-reboot-info`](https://github.com/rackerlabs/cs-reboot-info)
+tool to discover the reboot windows of servers across all regions at once.
+Downloads for many platforms are available
```
Comment (jnoller) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Binary Downloads

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md (hunk 4 above)
Comment (ycombinator) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Include small screenshot of region selector or is that too much detail for this post?

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    | Next Gen (ORD) | 0f8482b6  | My ORD Server | 03 Mar 16:00 - 03 Mar 18:00 | 03 Mar 10:00 - 03 Mar 12:00 |
+
+Additionally, a CSV file containing the details can be generated with
+the `--csv` option.
+
+    $ ./cs-reboot-info --csv <username> <api key>
+    Regions with a Cloud Servers endpoint: ORD, DFW, SYD, IAD, HKG
+    Found both First and Next Generation endpoints.
+    Writing output to cs-reboot-info-output.csv
+
```
Comment (jnoller) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Going to need to update this with the new output format and csv stuff goody added

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md (hunk 3 above)
Comment (jnoller) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Want to add a link to the issue tracker

Comment (briancurtin) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Yes - is there another or more preferred way? I couldn't get anything else to work in Mou and I don't normally do markdown, so this was all I could get working.

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+For more information, see https://github.com/rackerlabs/cs-reboot-info
```
Comment (ycombinator) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: To be safe, you could just use `[https://github.com/rackerlabs/cs-reboot-info](https://github.com/rackerlabs/cs-reboot-info)`

**Comment by ycombinator** [comment]
LGTM. Merging.

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    - devex
+    - utils
+---
+
+In the wake of
+[recently announced vulnerabilities](http://xenbits.xen.org/xsa/) to the
+Xen hypervisor that our Cloud Servers platform is built on top of, a reboot
+will be necessary in some instances on both our First Generation and Next
+Generation Cloud Servers. The details of our announcement are available at
+[https://community.rackspace.com/general/f/53/t/4978]() and via
```
Comment (kenperkins) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Needs URL

File: src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+[https://status.rackspace.com/]().
```
Comment (kenperkins) [inline src/site_source/_posts/2015-02-28-cloud-servers-reboot-info.md]: Needs url in ()

**Pull request merged**